lib/spack/spack/util/gpg.py

```diff
@@ -62,7 +63,10 @@ def init(gnupghome=None, force=False):
 
     # Set the executable objects for "gpg" and "gpgconf"
     with spack.bootstrap.ensure_bootstrap_configuration():
-        spack.bootstrap.ensure_gpg_in_path_or_raise()
+        if sys.platform == "win32":
+            spack.bootstrap.win_ensure_or_acquire_resource("gpg")
+        else:
+            spack.bootstrap.ensure_gpg_in_path_or_raise()
```

Comment:
(minor) I'm thinking it might be nice to move this `if` inside `spack.bootstrap` and have a higher-level `give-me-gpg` function (apologies if this has been this way for a while and I'm just now asking about it - although I think this became clearer to me as you reorganized/improved things).

Reply:
Makes a lot of sense, will address.